src/components/button/button.scss

```diff
@@ -33,7 +33,12 @@
   font-weight: font.$weight-semibold;
   height: sizes.$control-height; /* 3 */
   line-height: normal;
-  padding: sizes.$control-pad-vertical sizes.$control-pad-horizontal;
+  /* @TODO - restore original, below */
```

Comment:
This file should be completely restored before approval and merge.

Reply:
Now that #651 has been merged, you should have some properties to test without needing to introduce this code.

Reply:
Right you are. I'll revert that file now.